assignment2/inheritance.js

```diff
@@ -0,0 +1,21 @@
+function employee (nameS)
+{
+    this.name=nameS;
+    
+}
+//employee.prototype.getname=function()
```

Comment:
@psinha20 Remove the console messages. Please indent code properly before creating the PR and remove unnecessary blank lines.